core-viewmodel/src/test/java/com/rocket/android/core/viewmodel/ExampleUnitTest.kt

```diff
@@ -1,17 +1,17 @@
 package com.rocket.android.core.viewmodel
 
+import org.junit.Assert.assertEquals
 import org.junit.Test
 
-import org.junit.Assert.*
-
 /**
  * Example local unit test, which will execute on the development machine (host).
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class ExampleUnitTest {
+
```

Comment:
ExampleTests default file out if you are not using them

Reply:
I have removed all example tests.